### PR TITLE
fix(file-icons): make csv and tsv use spreadsheet icon

### PR DESCRIPTION
**Pull request by Multimo**
The current csv file icon uses the document icon. As per design, we should swap this to use the spreadsheet icon. 

# Screenshots:
### document
image

### Spreadsheet
image

### Before:
image

### After:
Doing the changes via pr here in local node_modules. 
image

### CODE REVIEW

**Review by tjuanitas — DISMISSED** [review]

**Comment by jstoffan** [comment]
Should this be updated in the `previewIcons` map, as well? Is there anywhere else this change is needed (PHP) to stay consistent across our applications?

**Comment by Multimo** [comment]
> Should this be updated in the `previewIcons` map, as well? Is there anywhere else this change is needed (PHP) to stay consistent across our applications?

Hey @jstoffan, looks like previewIcons is already done using the spreadsheet based on what I see here: https://github.com/box/box-ui-elements/blob/3109c4362b65ad2f00b7b5f35c97766b68202527/src/components/preview/previewIcons.ts#L149

With regards to the anywhere else, I am not sure sorry. If you have some idea of to where I should start looking please let me know.

**Review by jstoffan — DISMISSED** [review]
LGTM, but please make sure to update the PR title to reflect the latest changes prior to merging.

File: src/icons/file-icon/FileIcon.tsx [hunk 1]
```diff
@@ -166,7 +166,7 @@ const EXTENSIONS: { [key: string]: { [key: string]: string } } = {
     IconFilePresentation: mirror(['odp', 'otp', 'pot', 'potx']),
     IconFilePowerpointPresentation: mirror(['ppt', 'pptx', 'pptm']),
     IconFilePhotoshop: mirror(['psd']),
-    IconFileSpreadsheet: mirror(['ods', 'xlt', 'xltx']),
+    IconFileSpreadsheet: mirror(['csv', 'tsv', 'ods', 'xlt', 'xltx']),
```
Comment (tjuanitas) [inline src/icons/file-icon/FileIcon.tsx:169]: nit: the extensions should be sorted
```suggestion
    IconFileSpreadsheet: mirror(['csv', 'ods', 'tsv', 'xlt', 'xltx']),
```

**Review by jstoffan — APPROVED** [review]

**Comment by tjuanitas** [comment]
Force merging this due to the Semantic Pull Request service being down. It currently passes other checks.

**Pull request merged**